docs/spec.md

```diff
@@ -5516,6 +5511,87 @@ Produces a `result` tuple from values `val`.
 // %result: ([1.0, 2.0], (3))
 ```
 
+### uniform_dequantize
+
+#### Semantics
+
+Performs element-wise conversion of uniform quantized tensor `operand` to a
```

Comment:
I don't think we have defined what is a uniform quantized tensor. Replace with "quantized tensor" here, below and in Ops.td?

Reply:
done.